### PR TITLE
Website sends confirmation email to the user and to whichcard email 

### PR DESCRIPTION
**Pull request by Masuda50**
-- Also changed some input fields in the front end which were previously incorrect

### CODE REVIEW

**Comment by Masuda50** [comment]
Will push test for these when I figure out how too --

File: cards/views.py [hunk 1]
```diff
@@ -184,5 +187,42 @@ def about_us(request):
...
+            msg.content_subtype = "html"
+            msg.send()
+
+            form = FeedbackForm()
+            return render(request, 'cards/submit_feedback.html', {'form': form})
+
+
+    else:
+        form = FeedbackForm()
+        return render(request, 'cards/submit_feedback.html', {'form': form})
```
Comment (natharrow) [inline cards/views.py:228]: Looks good! Does it send emails to both us and the user?

File: cards/views.py [hunk 2]
```diff
@@ -184,5 +187,42 @@ def about_us(request):
...
+    if request.method == "POST":
+        form = FeedbackForm(request.POST)
+        if form.is_valid(): #if the form is valid get all values 
+            name = form.cleaned_data['name']
+            email = form.cleaned_data['email']
+            category = form.cleaned_data['category']
+            subject = form.cleaned_data['subject']
+            body = form.cleaned_data['body']
+            
+            MessageString = 'NAME: '+ name + "<br> " + 'EMAIL: ' +  email + "<br>" + 'SUBJECT: ' + subject+ '<br>' + 'BODY: '+ body
```
Comment (natharrow) [inline cards/views.py:199]: As Yorli mentioned, we can stick to the pythin my_variable naming convention.

Comment (Masuda50) [reply cards/views.py:199]: a habit from my java/c++ days -- will be more mindful next time

**Pull request merged**